easybuild/tools/package/package_naming_scheme/easybuild_pns.py

```diff
@@ -42,12 +41,6 @@ def name(self, ec):
         return '%s-%s' % (ec['name'], det_full_ec_version(ec))
 
     def version(self, ec):
-        """Determine package version: EasyBuild version used to build & install."""
-        ebver = str(EASYBUILD_VERSION)
-        if ebver.endswith('dev'):
-            # try and make sure that 'dev' EasyBuild version is not considered newer just because it's longer
-            # (e.g., 2.2.0 vs 2.2.0dev)
-            # cfr. http://rpm.org/ticket/56,
-            # https://debian-handbook.info/browse/stable/sect.manipulating-packages-with-dpkg.html (see box in 5.4.3)
-            ebver.replace('dev', '~dev')
-        return 'eb-%s' % ebver
+        """Determine package version: Date used while build & install."""
+        date_stamp = datetime.today().strftime('%Y%m%d')
```

Comment:
@boegel Is this what we want? I imagine we might still like to see the EB version here, since that holds a bit more useful information than a timestamp?

As I understood it, the problem was the version starting with `eb-`, could we just make append `~eb` instead?

Reply:
Hi, @ocaisa ! It make sense.
I was just thinking about my version with datestamp to have a clear look when module was built. And IMHO, from user perspective, datestamp in a version looks more useful rather than just eb version

Reply:
I agree with @ocaisa that it makes sense to keep tracking the EasyBuild version that was used to build the package.

I can see why a datestamp would be useful, but it's mostly meaningless (it doesn't give any information about what's inside the package, just when it was created).

Reply:
Ok. So I could revert some part of changes and come back to the version based on `ebver~eb`, where ebver determines as usual